prism/src/main/java/org/apache/falcon/service/EntitySLAMonitoringService.java

```diff
@@ -579,4 +624,21 @@ Date getInitialStartTime(Entity entity, String clusterName, String entityType) t
             return startTime.before(slaTime) ? startTime : slaTime;
         }
     }
+
+    public void updateProcessSLAMonitoring(String clusterName, String entityName, String date, String entityType) {
```

Comment:
Better to have the method name synonymous to what is there for feed makeProcessInstanceAvailable.